samples/client/petstore/cpp-tizen/src/UserManager.h

```diff
@@ -28,80 +28,80 @@ class UserManager {
  * \param accessToken The Authorization token. *Required*
  * \param userData The user data to be passed to the callback function.
  */
 bool createUserSync(char * accessToken,
-	User user, 
+	User body, 
 	
 	void(* handler)(Error, void* ) , void* userData);
 
 /*! \brief Create user. *Asynchronous*
  *
  * This can only be done by the logged in user.
- * \param user Created user object *Required*
+ * \param body Created user object *Required*
  * \param handler The callback function to be invoked on completion. *Required*
  * \param accessToken The Authorization token. *Required*
  * \param userData The user data to be passed to the callback function.
  */
 bool createUserAsync(char * accessToken,
-	User user, 
+	User body, 
 	
 	void(* handler)(Error, void* ) , void* userData);
 
 
 /*! \brief Creates list of users with given input array. *Synchronous*
  *
  * 
- * \param user List of user object *Required*
+ * \param body List of user object *Required*
  * \param handler The callback function to be invoked on completion. *Required*
  * \param accessToken The Authorization token. *Required*
  * \param userData The user data to be passed to the callback function.
  */
 bool createUsersWithArrayInputSync(char * accessToken,
-	std::list<std::list> user, 
+	std::list<User> body, 
 	
 	void(* handler)(Error, void* ) , void* userData);
 
 /*! \brief Creates list of users with given input array. *Asynchronous*
  *
  * 
- * \param user List of user object *Required*
+ * \param body List of user object *Required*
  * \param handler The callback function to be invoked on completion. *Required*
  * \param accessToken The Authorization token. *Required*
  * \param userData The user data to be passed to the callback function.
  */
 bool createUsersWithArrayInputAsync(char * accessToken,
-	std::list<std::list> user, 
+	std::list<User> body, 
 	
 	void(* handler)(Error, void* ) , void* userData);
 
 
 /*! \brief Creates list of users with given input array. *Synchronous*
  *
  * 
- * \param user List of user object *Required*
+ * \param body List of user object *Required*
  * \param handler The callback function to be invoked on completion. *Required*
  * \param accessToken The Authorization token. *Required*
  * \param userData The user data to be passed to the callback function.
  */
 bool createUsersWithListInputSync(char * accessToken,
-	std::list<std::list> user, 
+	std::list<User> body, 
 	
 	void(* handler)(Error, void* ) , void* userData);
 
 /*! \brief Creates list of users with given input array. *Asynchronous*
  *
  * 
- * \param user List of user object *Required*
+ * \param body List of user object *Required*
  * \param handler The callback function to be invoked on completion. *Required*
  * \param accessToken The Authorization token. *Required*
  * \param userData The user data to be passed to the callback function.
  */
 bool createUsersWithListInputAsync(char * accessToken,
-	std::list<std::list> user, 
+	std::list<User> body, 
```

Comment:
yes that was not intentionally. Maybe i broke something there. I will check whats the cause for this

Reply:
This change occurred when i ran the `generate-samples.sh` for all generators that use cpp. I.e `bin/configs/cpp*` and `bin/configs/other/cpp*`. I got a little confused with the samples generator because he would give me many untracked files that didnt really make sense. I cloned my fork again and it looks better now. This change still occures when generating samples, but i guess thats because `cpp-tizen` hasnt been updated for a while